packages/dialtone-tokens/tokens/base/default.json

```diff
@@ -492,7 +492,7 @@
           "x": "{size.0}",
           "y": "{size.0}",
           "blur": "{size.0}",
-          "spread": "{size.300}",
+          "spread": "{size.200} + {size.100}",
```

Comment:
Should this be a specific size instead of a calculation? 

I remember we were discussing about this calculations on sizes but don't remember exactly what was the final decision on that discussion.

Reply:
This one is acceptable and is inline with how we math certain Design Tokens, and isn't necessarily a magic number. In this case it is giving the appearance of `2px` while technically being `3px`. We definitely don't want to create a Design Token for `3px` since that size is against the grain of the size/space system.